kcl/kcl.go

```diff
@@ -275,7 +277,6 @@ func (kclp *KCLProcess) Run() {
 		err = kclp.handleLine(line)
 		if err != nil {
 			kclp.ioHandler.writeError(fmt.Sprintf("ERR Handle line: %+#v", err))
-			return
```

Comment:
Based on the [python implement](Encountered shutdown exception, skipping checkpoint) it seems like most errors shouldn't cause the process to exit.